bug: ignore comments don't work

repro

1. `git clone https://github.com/gcpug/zagane.git`
2. `cd zagane`
3. edit test code as follows.
```diff
diff --git a/passes/unstopiter/testdata/src/a/a.go b/passes/unstopiter/testdata/src/a/a.go
index 0ad4cf5..c78dd04 100644
--- a/passes/unstopiter/testdata/src/a/a.go
+++ b/passes/unstopiter/testdata/src/a/a.go
@@ -8,7 +8,8 @@ import (

 func f1(ctx context.Context, client *spanner.Client) {
        stmt := spanner.Statement{SQL: `SELECT 1`}
-       _, _ = client.Single().Query(ctx, stmt).Next() // want "iterator must be stopped"
+       //lint:ignore unstopiter reason
+       _, _ = client.Single().Query(ctx, stmt).Next()
        client.Single().Query(ctx, stmt).Stop()        // OK
        defer client.Single().Query(ctx, stmt).Stop()  // OK
 }
```
4. `go test .\passes\unstopiter\unstopiter_test.go`

Expected: Test passes
Actual: Test fails

```sh
$ go test .\passes\unstopiter\unstopiter_test.go
--- FAIL: Test (2.40s)
    analysistest.go:251: a/a.go:12:30: unexpected diagnostic: iterator must be stopped
FAIL
FAIL    command-line-arguments  2.550s
```

This is maybe `github.com/gostaticanalysis/comment` bug or misusage.

--- a/go.mod
+++ b/go.mod
@@ -4,7 +4,7 @@ go 1.12
 
 require (
 	github.com/gostaticanalysis/analysisutil v0.0.0-20190329151158-56bca42c7635
-	github.com/gostaticanalysis/comment v1.2.0
+	github.com/gostaticanalysis/comment v1.3.0
 	golang.org/x/crypto v0.0.0-20190325154230-a5d413f7728c // indirect
 	golang.org/x/net v0.0.0-20190328230028-74de082e2cca // indirect
 	golang.org/x/sys v0.0.0-20190329044733-9eb1bfa1ce65 // indirect

--- a/passes/unclosetx/testdata/src/a/a.go
+++ b/passes/unclosetx/testdata/src/a/a.go
@@ -12,6 +12,8 @@ func f1(ctx context.Context, client *spanner.Client) {
 	tx := client.ReadOnlyTransaction()   // OK
 	tx.Close()
 	client.Single() // OK
+	client.ReadOnlyTransaction()         //lint:ignore zagane OK
+	client.ReadOnlyTransaction()         //lint:ignore unclosetx OK
 }
 
 func f2(ctx context.Context, client *spanner.Client) {

--- a/passes/unclosetx/unclosetx.go
+++ b/passes/unclosetx/unclosetx.go
@@ -62,10 +62,11 @@ func run(pass *analysis.Pass) (interface{}, error) {
 		for _, b := range f.Blocks {
 			for i := range b.Instrs {
 				pos := b.Instrs[i].Pos()
+				line := pass.Fset.File(pos).Line(pos)
 
 				// skip
-				if cmaps.IgnorePos(pos, "zagane") ||
-					cmaps.IgnorePos(pos, "unclosetx") ||
+				if cmaps.IgnoreLine(pass.Fset, line, "zagane") ||
+					cmaps.IgnoreLine(pass.Fset, line, "unclosetx") ||
 					isSingle(b.Instrs[i], single) {
 					continue
 				}

--- a/passes/unstopiter/testdata/src/a/a.go
+++ b/passes/unstopiter/testdata/src/a/a.go
@@ -11,6 +11,8 @@ func f1(ctx context.Context, client *spanner.Client) {
 	_, _ = client.Single().Query(ctx, stmt).Next() // want "iterator must be stopped"
 	client.Single().Query(ctx, stmt).Stop()        // OK
 	defer client.Single().Query(ctx, stmt).Stop()  // OK
+	_, _ = client.Single().Query(ctx, stmt).Next() //lint:ignore zagane OK
+	_, _ = client.Single().Query(ctx, stmt).Next() //lint:ignore unstopiter OK
 }
 
 func f2(ctx context.Context, client *spanner.Client) {

--- a/passes/unstopiter/unstopiter.go
+++ b/passes/unstopiter/unstopiter.go
@@ -58,10 +58,11 @@ func run(pass *analysis.Pass) (interface{}, error) {
 		for _, b := range f.Blocks {
 			for i := range b.Instrs {
 				pos := b.Instrs[i].Pos()
+				line := pass.Fset.File(pos).Line(pos)
 
 				// skip
-				if cmaps.IgnorePos(pos, "zagane") ||
-					cmaps.IgnorePos(pos, "unstopiter") {
+				if cmaps.IgnoreLine(pass.Fset, line, "zagane") ||
+					cmaps.IgnoreLine(pass.Fset, line, "unstopiter") {
 					continue
 				}
 

--- a/passes/wraperr/testdata/src/a/a.go
+++ b/passes/wraperr/testdata/src/a/a.go
@@ -129,3 +129,27 @@ func f10(ctx context.Context, client *spanner.Client) {
 		return outsideErr
 	})
 }
+
+func f11(ctx context.Context, client *spanner.Client) {
+	client.ReadWriteTransaction(ctx, func(ctx context.Context, txn *spanner.ReadWriteTransaction) error {
+		stmt := spanner.Statement{SQL: `SELECT 1`}
+		_, err := client.Single().Query(ctx, stmt).Next()
+		if err != nil {
+			//lint:ignore zagane OK
+			return wrap(err)
+		}
+		return nil
+	})
+}
+
+func f12(ctx context.Context, client *spanner.Client) {
+	client.ReadWriteTransaction(ctx, func(ctx context.Context, txn *spanner.ReadWriteTransaction) error {
+		stmt := spanner.Statement{SQL: `SELECT 1`}
+		_, err := client.Single().Query(ctx, stmt).Next()
+		if err != nil {
+			//lint:ignore wraperr OK
+			return wrap(err)
+		}
+		return nil
+	})
+}

--- a/passes/wraperr/wraperr.go
+++ b/passes/wraperr/wraperr.go
@@ -60,9 +60,10 @@ func (r *runner) run(pass *analysis.Pass) (interface{}, error) {
 					continue
 				}
 
-				if pos := r.wrapped(instr); pos != token.NoPos {
-					if !cmaps.IgnorePos(pos, "zagane") &&
-						!cmaps.IgnorePos(pos, "wraperr") {
+				if pos := r.wrapped(instr); pos.IsValid() {
+					l := pass.Fset.File(pos).Line(pos)
+					if !cmaps.IgnoreLine(pass.Fset, l, "zagane") &&
+						!cmaps.IgnoreLine(pass.Fset, l, "wraperr") {
 						pass.Reportf(pos, "must not be wrapped")
 					}
 				}

